.github/workflows/test.yml

```diff
@@ -77,7 +77,5 @@ jobs:
         cp config/application.yml.example config/application.yml
         gem install bundler --version $(tail -n1 Gemfile.lock)
         yarn install
-        RAILS_ENV=test bundle exec rails webpacker:compile
-        RAILS_ENV=test bin/rails db:create --trace
```

Comment:
Do we not need there anymore? Are they managed in `precompile` ?

Reply:
`RAILS_ENV=test bundle exec rails webpacker:compile` has never been required: it should always be "bin/rails assets:precompile")

`RAILS_ENV=test bin/rails db:create --trace` is managed by rspec